### PR TITLE
ZCS-12018: UI to display S3 buckets information

### PR DESCRIPTION
**Pull request by danielguarchajti**
Add ZaGlobalConfig to the initMethods array in order to be able to push the getS3BucketConfig and perform the request from the zimlet.

### CODE REVIEW

File: WebRoot/js/zimbraAdmin/globalconfig/model/ZaGlobalConfig.js [hunk 1]
```diff
@@ -27,13 +27,15 @@ ZaGlobalConfig = function() {
 	this.type = ZaItem.GLOBAL_CONFIG;
 //	this.attrsInternal = new Object();	
 	this.load();
+    this._init();
```
Comment (silentsakky) [inline WebRoot/js/zimbraAdmin/globalconfig/model/ZaGlobalConfig.js:30]: indentation

Comment (sakshirawat0247) [reply WebRoot/js/zimbraAdmin/globalconfig/model/ZaGlobalConfig.js:30]: done

**Review by lavegupta — APPROVED** [review]

File: WebRoot/js/zimbraAdmin/globalconfig/model/ZaGlobalConfig.js [hunk 2]
```diff
@@ -21,19 +21,21 @@
...
+    this.load();
+    this._init();
 }
 
 ZaGlobalConfig.prototype = new ZaItem;
 ZaGlobalConfig.prototype.constructor = ZaGlobalConfig;
 ZaItem.loadMethods["ZaGlobalConfig"] = new Array();
 ZaItem.modifyMethods["ZaGlobalConfig"] = new Array();
 ZaItem.modifyMethodsExt["ZaGlobalConfig"] = new Array();
+ZaItem.initMethods["ZaGlobalConfig"] = new Array();
```
Comment (lavegupta) [inline WebRoot/js/zimbraAdmin/globalconfig/model/ZaGlobalConfig.js:38]: Are we using this anywhere?

File: WebRoot/js/zimbraAdmin/globalconfig/model/ZaGlobalConfig.js [hunk 3]
```diff
@@ -21,19 +21,21 @@
...
-	this.type = ZaItem.GLOBAL_CONFIG;
-//	this.attrsInternal = new Object();	
-	this.load();
+ZaGlobalConfig = function () {
+    ZaItem.call(this, "ZaGlobalConfig");
+    this.attrs = new Object();
+    this.type = ZaItem.GLOBAL_CONFIG;
+    //this.attrsInternal = new Object();	
+    this.load();
+    this._init();
```
Comment (k-kato) [inline WebRoot/js/zimbraAdmin/globalconfig/model/ZaGlobalConfig.js:30]: Does `this` always have `_init` function in any case? I'm worried about an exception due to undefined function.

**Comment by rababkazi** [comment]
Closing this PR. `ZaItem.initMethods["ZaGlobalConfig"] = new Array();` and `this._init();` are not being used for Global Settings SM tab. Hence, they need not be added.

**Pull request closed without merging**